common/remote/sockfile.cpp

```diff
@@ -6906,14 +6933,34 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                 MAPCOMMAND(RFCgetinfo, cmdGetInfo);
                 MAPCOMMAND(RFCfirewall, cmdFirewall);
                 MAPCOMMANDCLIENT(RFCunlock, cmdUnlock, *client);
-                MAPCOMMANDCLIENT(RFCStreamRead, cmdStreamReadStd, *client);
-                MAPCOMMANDCLIENT(RFCStreamReadJSON, cmdStreamReadJSON, *client);
-                MAPCOMMANDCLIENTTESTSOCKET(RFCStreamReadTestSocket, cmdStreamReadTestSocket, *client);
                 MAPCOMMANDCLIENT(RFCcopysection, cmdCopySection, *client);
                 MAPCOMMANDCLIENTTHROTTLE(RFCtreecopy, cmdTreeCopy, *client, &slowCmdThrottler);
                 MAPCOMMANDCLIENTTHROTTLE(RFCtreecopytmp, cmdTreeCopyTmp, *client, &slowCmdThrottler);
                 MAPCOMMAND(RFCsetthrottle, cmdSetThrottle); // legacy version
                 MAPCOMMAND(RFCsetthrottle2, cmdSetThrottle2);
+                // row service commands
+                case RFCStreamRead:
+                {
+                    if (!dafilesrvRowServiceCmds && !client->isRowServiceClient())
```

Comment:
it seems the test for !dafilesrvRowServiceCmds && !client->isRowServiceClient() and the exception in these three cases is common (unless I missed something), why not extract it out before the switch case?

Reply:
it's common to these 3 case statements, so moving before the switch statement would require either another case statement of test of the cmd to see if it is one of these 3 before testing this expression.
But I'll move these 2 lines into a common method.